payments-core/src/test/java/com/stripe/android/StripeKtxTest.kt

```diff
@@ -264,13 +265,6 @@ internal class StripeKtxTest {
             stripe::retrievePaymentIntent
         )
 
-    @Test
-    fun `When repository returns null then retrievePaymentIntent should throw InvalidRequestException`(): Unit =
-        `Given repository returns null when calling retrieveAPI with String param then throws InvalidRequestException`(
```

Comment:
We no longer return `null` from these methods, so these tests don’t make sense anymore.